src/Header.js

```diff
@@ -1,18 +1,18 @@
 import React, { Component } from 'react';
 
+
 class Header extends Component {
 
   render() {
     return (
       <div className="Header">
-        <h1>Header Component</h1>
+        {/*<h1>Header Component</h1>*/}
```

Comment:
NIT
Let's not include commented out code. 
If its unnecessary delete it, if you need it for reference, either change it to a properly syntax comment or just don't commit it.